app/src/components/LogoutButton.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { View, Platform, ToastAndroid } from 'react-native';
+import { ListItem, Dialog, Text, Icon } from '@rneui/themed';
+import { useNavigation } from '@react-navigation/native';
+import { AuthState } from '../store';
+import { conn, query } from '../database';
+
+const db = conn.init;
+
+const LogoutButton = () => {
+  const [visible, setVisible] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const navigation = useNavigation();
+
+  const handleNoPress = () => {
+    setVisible(false);
+  };
+
+  const handleError = () => {
+    if (Platform.OS === 'android') {
+      ToastAndroid.show('Logout Failed: Please try again later', ToastAndroid.LONG);
+    }
+  };
+
+  const handleYesPress = () => {
+    const table = 'sessions';
+    const clearQuery = query.clear(table);
```

Comment:
Please create truncate query instead of just clearing the session.

Reply:
Hi mas @dedenbangkit actually clear is truncate query
https://github.com/akvo/nmis-mobile/blob/main/app/src/database/query.js#L25-L27

Reply:
Sorry, I mean the truncate should support multiple tables